src/rgw/rgw_http_client.cc

```diff
@@ -1151,6 +1151,7 @@ void *RGWHTTPManager::reqs_thread_entry()
           case CURLE_OPERATION_TIMEDOUT:
             dout(0) << "WARNING: curl operation timed out, network average transfer speed less than " 
               << cct->_conf->rgw_curl_low_speed_limit << " Bytes per second during " << cct->_conf->rgw_curl_low_speed_time << " seconds." << dendl;
+            break;
```

Comment:
looks to me like an intwentional fallthrough.
it has a case for a specific error, and then the "default" case is general error handling